extensions/jms-spi/deployment/src/main/java/io/quarkus/jms/spi/deployment/ConnectionFactoryWrapperBuildItem.java

```diff
@@ -0,0 +1,25 @@
+package io.quarkus.jms.spi.deployment;
+
+import java.util.function.Function;
+
+import javax.jms.ConnectionFactory;
+
+import org.wildfly.common.Assert;
```

Comment:
We definitely don't want to use a WildFly class here.

Reply:
OK, I remove it.